bin/install-platform

```diff
@@ -6,7 +6,7 @@ set -x
 # The path to the working directory - the root of the repo
 GIT_ROOT="$(git -C "${0%/*}" rev-parse --show-toplevel)"
 HELM_CHART_PATH=${HELM_CHART_PATH:-$GIT_ROOT}
-RELEASE_NAME="${RELEASE_NAME:-astronomer}"
```

Comment:
The name of the variable should remain `RELEASE_NAME`, we just want to change the value of this variable to all lowercase